cmd/agent/app/servers/thriftudp/transport.go

```diff
@@ -71,7 +73,7 @@ func createClient(destAddr, locAddr *net.UDPAddr) (*TUDPTransport, error) {
 // It will listen for incoming udp packets on the specified host/port
 // Example:
 // 	trans, err := thriftudp.NewTUDPClientTransport("localhost:9001")
-func NewTUDPServerTransport(hostPort string) (*TUDPTransport, error) {
+func NewTUDPServerTransport(hostPort string, bufferSize int) (*TUDPTransport, error) {
```

Comment:
instead of changing the API (which has a knock off effect on a lot of code), could we just add a SetSocketBufferSize method to TUDPTransport and call it from where we need it?  Especially since it's optional.